Update installation.md (#2817)
webpack/webpack.js.org: src/content/guides/installation.md

}
```

> T> To run the local installation of webpack you can access its bin version as `node_modules/.bin/webpack` or use npx, by running 'npx webpack' if you are using npm v5.2.0 or greater which will run the locally installed binary of webpack

Reviewer: ```suggestion
T> To run the local installation of webpack you can access its bin version as `node_modules/.bin/webpack`. Alternatively, if you are using npm v5.2.0 or greater, you can use npx by running 'npx webpack'. This will run the locally installed binary of webpack.
```

Reviewer 2: Makes sens, also can you change `bin` to `binary` for consistency in the 1st sentence

> T> To run the local installation of webpack you can access its bin version as `node_modules/.bin/webpack`. Alternatively, if you are using npm v5.2.0 or greater, you can use npx by running 'npx webpack'. This will run the locally installed binary of webpack.

Reviewer 2: ```suggestion
T> To run the local installation of webpack you can access its binary version as `node_modules/.bin/webpack`. Alternatively, if you are using npm v5.2.0 or greater, you can run 'npx webpack' to do it.
```

Reviewer 2: This is shorter, and not saying twice that we are going to run local binary by it

Reviewer 2: how was this resolved @lkounadis or argued against?

Author: I believe that it is a bit more elaborate as is tbh because it explains after what npx does.

Reviewer 2: I do not agree that repeating it twice in the same context (think: hint block) is a good idea, we can wait for more opinions but i am not approving this
